elements/map/src/generate.ts

```diff
@@ -50,9 +83,8 @@ export function createLayer(layer: EoxLayer, group?: string): olLayers.Layer {
       }),
     }),
     style: undefined, // override layer style, apply style after
-    // @ts-ignore
     ...(layer.type === "Group" && {
-      layers: layer.layers.reverse().map((l) => createLayer(l, layer.id)),
+      layers: layer.layers.map((l) => createLayer(l, layer.id)),
```

Comment:
Removing the `reverse()` here will reverse all the current map collections, right? Is there a reason for doing that?

Reply:
this was a mistake, trying to fix the `groupLayer`-Test. I was not aware that the EOX-Map expects the layer order in the opposite way of ol, i will re-add the `reverse` and update the groupLayer-Test